session/session.go

```diff
@@ -225,6 +225,8 @@ type session struct {
 
 	// indexUsageCollector collects index usage information.
 	idxUsageCollector *handle.SessionIndexUsageCollector
+
+	builtinFunctionUsage telemetry.BuiltinFunctionsUsage
```

Comment:
This map should be cleared periodically too.

Reply:
Fixed.

Also, I found we can do the collect work in `session.Close` to reduce lock acquiring.